src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs

```diff
@@ -50,6 +51,13 @@ internal abstract class NodeProviderOutOfProcBase
         /// </summary>
         private const int TimeoutForNewNodeCreation = 30000;
 
+        /// <summary>
+        /// The number of MSBuild processes allowed on the computer. 5 is (low priority/normal priority) * (administrator/not administrator) + (no node reuse).
```

Comment:
This is potentially low (in the pathological running all-possible-builds case), because it doesn't take TaskHost nodes into account.